packages/mwp-app-route-plugin/src/route.js

```diff
@@ -12,7 +15,13 @@ export const onPreResponse = {
 	 */
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
-
+		if (request.query) {
+			const trackingParam = request.query['_xtd']
+			if (trackingParam) {
+				response.header(EXTERNAL_TRACK_HEADER, trackingParam);
```

Comment:
this behavior definitely needs some documentation so that people can understand the intent in the future - what is being tracked? where does the _xtd param come from and what is its expected value/format? Maybe just a link to documentation elsewhere, but otherwise it's kinda mystery meat.